src/Elastic.Apm.EntityFrameworkCore/EfCoreDiagnosticListener.cs

```diff
@@ -38,7 +39,8 @@ public void OnNext(KeyValuePair<string, object> kv)
 				case { } k when k == RelationalEventId.CommandExecuting.Name && _agent.Tracer.CurrentTransaction != null:
 					if (kv.Value is CommandEventData commandEventData)
 					{
-						var newSpan = _agent.TracerInternal.DbSpanCommon.StartSpan(_agent, commandEventData.Command, InstrumentationFlag.EfCore);

```

Comment:
I'd prefer keeping the call to `DbSpanCommon.StartSpan` - that way all db spans are started in the same method. And then I'd just add an optional parameter to it and pass `true` so it calls `StartSpanOnCurrentExecutionSegment` with `captureStackTraceOnStart: true`

Reply:
Alright, I didn't want to mess with the internal API and HTTP calls use the same method, but I'll change it around then